OBJECT/CreateAmbientPickup.md

```diff
@@ -25,3 +26,8 @@ Pickup hashes: pastebin.com/8EuSv2r1
 * **p8**: 
```

Comment:
Take a look at the code I posted, I was able to figure out `p8` is `netMissionEntity` by looking at the native code for [`CREATE_VEHICLE`](https://docs.fivem.net/natives/?_0xDD75460A) in the game's binary.